WordPress/Sniffs/XSS/EscapeOutputSniff.php

```diff
@@ -274,19 +274,41 @@ public function process( PHP_CodeSniffer_File $phpcsFile, $stackPtr )
 			// Now check that next token is a function call.
 			if ( T_STRING === $this->tokens[ $i ]['code'] ) {
 
+				$ptr = $i;
+
 				$functionName = $this->tokens[ $i ]['content'];
+
+				$function_opener = $this->phpcsFile->findNext( array( T_OPEN_PARENTHESIS ), $i + 1, null, null, null, true );
+
 				$is_formatting_function = isset( self::$formattingFunctions[ $functionName ] );
 
-				// Skip pointer to after the function.
-				if ( $_pos = $this->phpcsFile->findNext( array( T_OPEN_PARENTHESIS ), $i, null, null, null, true ) ) {
+				if ( $function_opener ) {
+
+					if ( 'array_map' === $functionName ) {
+
+						// Get the first parameter (name of function being used on the array).
+						$mapped_function = $this->phpcsFile->findNext(
+							PHP_CodeSniffer_Tokens::$emptyTokens
+							, $function_opener + 1
+							, $tokens[ $function_opener ]['parenthesis_closer']
+							, true
```

Comment:
Oops! I actually copied this from another part of the code (which I must have written :smile:), but I'll correct it.